steampipecloud/connection_config.go

```diff
@@ -59,12 +60,39 @@ func connect(_ context.Context, d *plugin.QueryData) (*openapiclient.APIClient,
 	}
 
 	if host != "" && !strings.Contains(host, "cloud.steampipe.io") {
-		configuration.Servers = []openapiclient.ServerConfiguration{
-			{
-				URL:         fmt.Sprintf("%s/api/v0", host),
-				Description: "Local API",
-			},
+		parsedURL, parseErr := url.Parse(host)
```

Comment:
@siddharthaturbot Was there a specific scenario that prompted this change? I couldn't find anything in the PR description or comments about it.